es6-classes/circle.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(radius) {
+    super(20, 60);
```

Comment:
Instead of using hard coded values, we want this to be reusable. We should be utilizing our `width` property to pass in what the perimeter and area are.